plugin-jdbc-rockset/src/main/java/io/kestra/plugin/jdbc/rockset/Query.java

```diff
@@ -50,16 +51,19 @@
 public class Query extends AbstractJdbcQuery implements RunnableTask<AbstractJdbcQuery.Output>, AutoCommitInterface {
     protected final Boolean autoCommit = true;
 
+    @PluginProperty(dynamic = true)
```

Comment:
Make a interface to have a common documenation

Reply:
done